Makefile

```diff
@@ -0,0 +1,51 @@
+#==================
+# Macros
+#==================
+
+pardir = $(patsubst %/,%,$(dir $(abspath $(1))))
+
+
+#==================
+# Helper Constants
+#==================
+
+THIS := $(abspath $(lastword $(MAKEFILE_LIST)))
+HERE := $(call pardir,$(THIS))
+DRAW := $(HERE)/draw_party
+
+
+#==================
+# Targets
+#==================
+
+.PHONY: clean
+clean:
+	cd $(DRAW)/client && npm run clean
```

Comment:
put `@` before all of the cd commands because I don't want to see them.

Reply:
That also silences the actual command though so I'd be in favor of just leaving it